pkg/cover/backend/elf.go

```diff
@@ -249,7 +292,11 @@ func readTextRanges(file *elf.File) ([]pcRange, []*CompileUnit, error) {
 					}
 				}
 			}
-			ranges = append(ranges, pcRange{r[0], r[1], unit})
+			if module.Name == "" {
```

Comment:
Since Addr==0 for kernel, I think this is unnecessary as well.

Reply:
Same comment as above, it's needed